README.md

```diff
@@ -256,6 +256,12 @@ class MyComponent
 end
 ```
 
+### `init_label_set`
+
+The time series of a metric are not initialized until something happens. For counters, for example, this means that the time series do not exist until the counter is incremented for the first time.
+
+To get around this problem the client provides the `init_label_set` method that can be used to initialise the time serie of a metric for a given label set.
```

Comment:
/s/serie/series/

(series is one of those weird words that is both singular and plural, like fish. English...)

Reply:
#til